src/main/java/org/assertj/core/api/AbstractMapAssert.java

```diff
@@ -99,6 +99,39 @@ public SELF allSatisfy(BiConsumer<? super K, ? super V> entryRequirements) {
     return myself;
   }
 
+  /**
+   * Verifies that at least one map entry satisfies the given {@code entryRequirements} .
+   * <p>
+   * Example:
+   * <pre><code class='java'> Map&lt;TolkienCharacter, Ring&gt; elvesRingBearers = new HashMap&lt;&gt;();
+   * elvesRingBearers.put(galadriel, nenya);
+   * elvesRingBearers.put(gandalf, narya);
+   * elvesRingBearers.put(elrond, vilya);
+   *
+   * // this assertion succeeds:
+   * assertThat(elvesRingBearers).anySatisfy((character, ring) -&gt; {
+   *   assertThat(ring).isIn(vilya);
```

Comment:
add `oneRing` and `nibelungRing` to `isIn` arguments otherwise it looks a bit weird

Reply:
I agree, yet I would resolve it another way - checking a map one any of vilya, oneRing and nibelungRing is not quite a meaningful test.

What about `isEqualTo`?